src/modules/ZFCPController.rb

```diff
@@ -52,7 +52,7 @@ def main
 
       @controllers = nil
 
-      @activated_controllers = {}
+      @activated_controllers = nil
```

Comment:
is it needed at all?

Reply:
Yes, it is needed for testing as modules are kind of singleton. So calling `main` will clean-up the cached values.